WorldSvr/OnWarehouse.c

```diff
@@ -65,10 +97,6 @@ CLIENT_PROCEDURE_BINDING(WAREHOUSE_CURRENCY_DEPOSIT) {
 
 	SocketSend(Socket, Connection, Response);
 	return;
-
-error:
```

Comment:
You can replace this error: label with your SocketSend(Socket, Connection, Response); with Result = 0 so you can keep using goto error.